extension/js/common/browser/browser.ts

```diff
@@ -55,13 +56,14 @@ export class Browser {
   public static openSettingsPage = async (path = 'index.htm', acctEmail?: string, page = '', rawPageUrlParams?: Dict<UrlParam>, addNewAcct = false) => {
     const basePath = chrome.runtime.getURL(`chrome/settings/${path}`);
     const pageUrlParams = rawPageUrlParams ? JSON.stringify(rawPageUrlParams) : undefined;
+    const preferedLinkOpener = Catch.browser().name === 'thunderbird' ? BgUtils : Browser;
```

Comment:
I think it'll be simpler to add this check to `Browser.openExtensionTab`, so it'll look like this:
```ts
public static openExtensionTab = (url: string) => {
  if (Catch.browser().name === 'thunderbird') {
    BgUtils.openExtensionTab(url)
  } else {
    ...
  }
}
```

Reply:
Thanks Roma - your suggestion is way better. Will update the changes.